docs/release-notes/metrics-overwrites.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Improvements**
+
+-  API: It is now allowed to report duplicate metrics across multiple ``report_metrics`` with the
+   same ``steps_completed`` if their value is the same.
```

Comment:
suggestion:

**Improvements**

-  API: Allow the reporting of duplicate metrics across multiple ``report_metrics`` with the
   same ``steps_completed``, provided they have identical values.